src/laravel/pagseguro/Sender/Sender.php

```diff
@@ -144,9 +144,9 @@ public function setPhone($phone)
     {
         if ($phone === null) {
             $phone = [];
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method setPhone uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/michaeldouglas010790/laravel-pagseguro/pullRequest?prid=5037054)